src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -318,6 +319,39 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
     {
         emitSize = EA_SCALABLE;
         opt      = emitter::optGetSveInsOpt(emitTypeSize(intrin.baseType));
+
+        switch (intrin.id)
```

Comment:
can you please revert the merge from my PR at https://github.com/dotnet/runtime/pull/105348 and merge `main`. These changes were present earlier in my PR but were removed later.